segmenters/nlp/SlidingWindowSegmenter/README.md

```diff
@@ -1,3 +1,62 @@
 # SlidingWindowSegmenter
 
-split the text on the doc-level into overlapping substrings on the chunk-level. 
\ No newline at end of file
+SlidingWindowSegmenter is one segmenter used in nlp domain. It splits the text on the doc-level into overlapping substrings on the chunk-level. 
+
+##Usage:
```

Comment:
Leave a space between the hashtag and the title pls. Otherwise the markdown is not displayed correctly.

Reply:
✅  thanks!